html/feed/feed.js

```diff
@@ -345,16 +345,27 @@ container.appendChild (card);
 
 
 
+const filterBtnTag = document.querySelector ("#tag")
 
+filterBtnTag.addEventListener ('click', async function (){
+  try {
+    const response = await fetch (`${baseUrl}/social/posts?/auction/listings?_tag=my_tag&_active=true`,headerOptions); 
```

Comment:
its good that you include the correct querys in the fetch but remember to set them to true so it acualy shows in the response. also here you are fetching social then auction? those are two different things
we can do a live session next week if you get stuck : )